js/src/pages/price-benchmark/product-comparison-chart.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import usePriceBenchmarkSummary from '~/hooks/usePriceBenchmarkSummary';
+import HorizontalStackedBar from '~/components/horizontal-stacked-bar';
+
+/**
+ * ProductComparisonChart component.
+ *
+ * This component renders a horizontal stacked bar chart that compares the pricing
+ * of the user's products to competitors. It uses data from the price benchmark summary
+ * to display segments representing different price categories.
+ *
+ * @return {JSX.Element|null} The rendered HorizontalStackedBar component or null if
+ * the data is not yet resolved or unavailable.
+ */
+const ProductComparisonChart = () => {
+	const { summary, hasFinishedResolution } = usePriceBenchmarkSummary();
+
+	if ( ! hasFinishedResolution || ! summary ) {
+		return null;
+	}
+
+	const segments = [
+		{
+			id: 'similar',
+			label: __( 'Similar', 'google-listings-and-ads' ),
+			value: summary.price_similar,
+			color: '#F0B849',
```

Comment:
Probably overkill for now, but if we find that we're reusing these hex values in other places, we should consider moving them to `js/src/constants.js`.